KitchenService/RabbitReceiver.cs

```diff
@@ -48,13 +48,15 @@ public Task StopAsync(CancellationToken cancellationToken)
         //    await _orderHub.Clients.All.SendAsync("new-order", order);
         //}
         [RabbitListener(Binding = "waffle.binding")]
-        private void DoStuff()
+        public void DoStuff(string orderMessage)
```

Comment:
This needs to be public. I could not get the message converting to automatically happen unlike with streams - so I am looking into that.